kube/services/jobs/fence-visa-update-cronjob.yaml

```diff
@@ -0,0 +1,139 @@
+    # DEPRECATED! Remove when all commons are no longer using local_settings.py
+    #             for fence.
+    # -----------------------------------------------------------------------------
+              - name: "creds-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/creds.json"
+                subPath: creds.json
+              - name: "old-config-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/local_settings.py"
+                subPath: local_settings.py
+              - name: "config-helper"
+                readOnly: true
+                mountPath: "/var/www/fence/config_helper.py"
+                subPath: config_helper.py
+              - name: "json-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_credentials.json"
+                subPath: fence_credentials.json
+    # -----------------------------------------------------------------------------
+              - name: "yaml-merge"
+                readOnly: true
+                mountPath: "/var/www/fence/yaml_merge.py"
+                subPath: yaml_merge.py      
+              - name: "config-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence-config-secret.yaml"
+                subPath: fence-config.yaml
+              - name: "fence-google-app-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_app_creds_secret.json"
+                subPath: fence_google_app_creds_secret.json
+              - name: "fence-google-storage-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_storage_creds_secret.json"
+                subPath: fence_google_storage_creds_secret.json          
+            command: ["/bin/bash" ]
+            args:
+              - "-c"
+              - |
+                echo "${FENCE_PUBLIC_CONFIG:-""}" > "/var/www/fence/fence-config-public.yaml"         
+                python /var/www/fence/yaml_merge.py /var/www/fence/fence-config-public.yaml /var/www/fence/fence-config-secret.yaml > /var/www/fence/fence-config.yaml
+                
+                create_command="fence-create update-visas"
+
+                if [[ -z "$CHUNK_SIZE" ]]; then
+                    create_command+=" --chunk-size $CHUNK_SIZE"
+                fi
+                if [[ -z "THREAD_POOL_SIZE" ]]; then
+                    create_command+=" --thread-pool-size $THREAD_POOL_SIZE"
+                fi
+                if [[ -z "BUFFER_SIZE" ]]; then
+                    create_command+=" --buffer-size $BUFFER_SIZE"
+                fi
+                if [[ -z "CONCURRENCY" ]]; then
+                    create_command+=" --concurrency $CONCURRENCY"
+                fi
+
+                $create_command
+
+                if [[ $? != 0 ]]; then
+                  echo "Warning: non zero exit code: $?"
```

Comment:
don't just print a warning; you can do something like [this](https://github.com/uc-cdis/cloud-automation/blob/92e00e33dc51c7fb997914d7d9c9965d25a9b0c0/kube/services/jobs/etl-cronjob.yaml#L92-L93)